packages/ember-simple-auth/tests/unit/authenticators/oauth2-password-grant-test.js

```diff
@@ -333,17 +333,23 @@ describe('OAuth2PasswordGrantAuthenticator', () => {
 
       describe('when a refresh token is set', function() {
         it('sends an AJAX request to invalidate the refresh token', async function() {
+          let revokeRequests = [];
           server.post('/revoke', (request) => {
             let { requestBody } = request;
             let body = parsePostData(requestBody);
-
-            expect(body).to.eql({
-              'token_type_hint': 'refresh_token',
-              'token': 'refresh token!'
-            });
+            revokeRequests.push(body);
           });
 
           await authenticator.invalidate({ 'access_token': 'access token!', 'refresh_token': 'refresh token!' });
+          expect(revokeRequests.length).to.eq(2);
+          expect(revokeRequests.find(body => body.token_type_hint === 'refresh_token')).to.eql({
+            'token_type_hint': 'refresh_token',
+            'token': 'refresh token!'
+          });
```

Comment:
I assume it would be simpler to just wrap the [`expect` in line 340](https://github.com/simplabs/ember-simple-auth/pull/2349/files#diff-eb0b12906e6123e788d141897aca52bfb626458de79e71dec56d390561510bc0L340) in an `if body.token_type_hint === 'refresh_token'`.

Sorry, I think that's what you had originally but I didn't fully understand the issue before…